wpilibc/src/main/native/cpp/sysid/SysIdRoutineLog.cpp

```diff
@@ -11,10 +11,7 @@
 using namespace frc::sysid;
 
 SysIdRoutineLog::SysIdRoutineLog(std::string_view logName)
-    : m_logName(logName),
-      m_state(wpi::log::StringLogEntry{
-          frc::DataLogManager::GetLog(),
-          fmt::format("sysid-test-state{}", logName)}) {
+    : m_logName(logName) {
   m_state.Append(StateEnumToString(State::kNone));
```

Comment:
Calling m_state.Append() before initializing m_state seems like a bug.

Reply:
Fixed, kNone isn't supposed to be logged until the entry is initialized